src/dashboards/videos/details.jsonnet

```diff
@@ -182,46 +143,7 @@ dashboard.new(
       timeField='timestamp'
     )
   ),
-  gridPos={ x: 12, y: 0, w: 6, h: 9 }
-)
-.addPanel(
-  graphPanel.new(
-    title='Daily views',
-    description=|||
-      A view is counted when the user has clicked the play button in the interface
-      in the first ${VIEW_COUNT_THRESHOLD} seconds of the video.
-    |||,
-    datasource=lrs,
-  ).addTarget(
-    elasticsearch.target(
-      datasource=lrs,
-      query='%(video_query)s AND verb.id:"%(verb_played)s" AND %(time)s:[0 TO $VIEW_COUNT_THRESHOLD]' % {
-        video_query: video_id_query,
-        verb_played: verb_id_played_value,
-        time: single_escape_string(result_extensions_time_field),
-      },
-      metrics=[
-        {
-          id: '1',
-          type: 'count',
-        },
-      ],
-      bucketAggs=[
-        {
-          id: 'date',
-          field: 'timestamp',
-          type: 'date_histogram',
-          settings: {
-            interval: '1d',
-            min_doc_count: '0',
-            trimEdges: '0',
-          },
-        },
-      ],
-      timeField='timestamp'
-    )
-  ),
-  gridPos={ x: 12, y: 9, w: 12, h: 9 }
+  gridPos={ x: 0, y: 1, w: 4.8, h: 6 }
```

Comment:
Instead of hard-coding position values, calculate it would be more user friendly. WDYT?